packages/blade/src/components/Box/Box.web.tsx

```diff
@@ -0,0 +1,7 @@
+import styled from 'styled-components';
+import type { BoxProps } from './Box.d';
+import getBoxStyles from './getBoxStyles';
+
+const Box = styled.div<BoxProps>(getBoxStyles);
```

Comment:
This will always add a dummy new element(`div` in this case) in the dom whenever wrapped around anything?

Reply:
Yes, there will be a div added. The idea is to use this as a wrapper in places and not as an enhancer.

Reply:
We can but lets avoid doing it unless we see valid use-cases

Reply:
what does as="tag" do?

Reply:
Renders the component using the html primitive specified, for example `<Box as="h1">`